### ISSUE TITLE
FromConfigRegistryAuthHandlerTest: Use `assertThrows` instead of deprecated `ExpectedException.none()` `@Rule`

### ISSUE THREAD

**Issue by rohanKanojia** [eclipse/jkube#858]
As pointed out in [JUnit Javadocs](https://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html#none()), `ExpectedException.none()` rule is deprecated. We should use `Assert.assertThrows` instead

https://github.com/eclipse/jkube/blob/5a1828c0ece30567fdcf89ce7668bb8c2003211d/jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandlerTest.java#L42-L43

We need to refactor `testFromPluginConfigurationFailed()`. Here is what you need to do:
- [x] Remove `ExpectedException.none()` `@Rule` and use `assertThrows`
- [x] Use AssertJ's `assertThat` instead of Hamcrest assertion for asserting exception message `org.hamcrest.CoreMatchers.containsString`

**Comment by himanshu007-creator** [comment]
Hi, I would like to work on this issue 😊

**Comment by soumyadeeptadas** [comment]
Hi, i have tried on this issue and refactored the deprecated functionalities.

Heres the link to the PR: [#858 fix ](https://github.com/eclipse/jkube/pull/858)

**Comment by rohanKanojia** [comment]
@soumyadeeptadas : Hi, This issue was already assigned to @himanshu007-creator . Why did you create a PR for this?

Anyway, there are some more places where `ExpectedException.none()` is being used in tests. We need to update these tests as well:
```bash
jkube : $ git grep -c "ExpectedException.none()"
jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/OpenShiftRegistryAuthHandlerTest.java:1
jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/SystemPropertyRegistryAuthHandlerTest.java:1
jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/access/KeyStoreUtilTest.java:1
jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/auth/ecr/DockerRegistryAuthHandlerTest.java:1
```

**Comment by manusa** [comment]
Fixed in #1042
